examples/Shellminator_multiple_ESP32_websocket/Shellminator_multiple_ESP32_websocket.ino

```diff
@@ -130,7 +130,7 @@ void webSocketEvent(uint8_t num, WStype_t type, uint8_t * payload, size_t length
     case WStype_TEXT:
 
       // Print to serial, just for debug.
-      Serial.printf("[%u] get Text: %s\n", num, payload);
+      Serial[ num ].printf("[%u] get Text: %s\n", num, payload);
```

Comment:
Hi jetpax!

Thank you for the PR! Can you explain a bit more, why is it necessary? As far as I know, The Serial object is an instance, not an array of Stream type. The purpose of this line was to log the disconnect event to the Serial port, not to the WebSocket instance.

I think you wanted to fix this line:
https://github.com/dani007200964/Shellminator/blob/b1df586aac48a9beb15529003ae767fd2aa546fc/examples/Shellminator_multiple_ESP32_websocket/Shellminator_multiple_ESP32_websocket.ino#L135-L140

I just quickly looked at it and it can be problematic. I think it should look like this:
```c++
// In the Shellminator-IO files, the WebSocket channel
// is implemented as a circular buffer. The incoming data
// from the clients has to be pushed to this circular buffer
// in the WebSocket event.
shell.webSocketPush( payload, length );
break;
```

If I'm wrong and I don't understand it correctly, please let me know. Also, it is not a big effort, can you please check the fix, that I showed? I'm a bit busy currently, and all of my setups are lying in the lab somewhere.

Also, if you need any help with this, please let me know.